runtime/compiler/net/Message.hpp

```diff
@@ -0,0 +1,379 @@
+/*******************************************************************************
+ * Copyright (c) 2020, 2020 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#ifndef MESSAGE_H
+#define MESSAGE_H
+
+#include <vector>
+#include <stdlib.h>
+#include "net/MessageBuffer.hpp"
+#include "net/MessageTypes.hpp"
+#include "OMR/Bytes.hpp" // for alignNoCheck
```

Comment:
I think including this file breaks the build because we have two unrelated definitions of `align()` in the codebase that clash.

The `align()` used here

https://github.com/eclipse/omr/blob/d3401cf86613f355d826a7b2aef301ae35fc437b/compiler/runtime/OMRCodeCache.hpp#L85-L86

refers to this implementation:

https://github.com/eclipse/omr/blob/d3401cf86613f355d826a7b2aef301ae35fc437b/compiler/runtime/Alignment.cpp#L30-L34

but since the use is not properly qualified once we include that header it will refer to `OMR::align` which is an incompatible function.

I'll submit a PR to OMR to fix it.

Reply:
I've opened https://github.com/eclipse/omr/pull/4943 to get it fixed.